pkg/controller/argocd/rolebinding.go

```diff
@@ -157,6 +157,15 @@ func (r *ReconcileArgoCD) reconcileClusterRoleBinding(name string, role *v1.Clus
 		roleBinding = newClusterRoleBindingWithname(name, cr)
```

Comment:
This like can be deleted.  (I know you didn't add it.)

Reply:
I think it is not redundant. When we do a `GET` on `clusterRoleBinding`, if it doesn't exist, it populates the `roleBinding` object with an empty struct, so we need to reinitialize it.